docs/components/concepts/messages.md

```diff
@@ -40,13 +48,22 @@ When a subscription is opened, it polls the buffer for a proper message. If a pr
 The buffering of a message is disabled when its TTL is set to zero. If no proper subscription is open, the message is discarded.
 
 <details>
-   <summary>Publish message with TTL via zbctl</summary>
+   <summary>Publish message with TTL via Camunda 8 REST API</summary>
    <p>
 
 ```
-zbctl publish message "Money collected" --correlationKey "order-123" --ttl 1h
+curl -L 'http://localhost:8080/v2/messages/publication' \
+-H 'Content-Type: application/json' \
+-H 'Accept: application/json' \
+-d '{
+  "name": "Money collected",
+  "correlationKey": "order-123",
+  "timeToLive": 3600000
```

Comment:
@pepopowitz @tmetzke - the API docs have TTL listed as "Timespan (in ms) to buffer the message on the broker." Is this correct? Is 1 hour (in milliseconds) a good realistic value?

Reply:
That is correct, as far as I know, yes. Realistic, well, classic Bernd answer: It depends 🤷 
There are scenarios where this makes sense and it was the example before already, so 🤷

Reply:
Resolving because that's a good enough answer for me! TY!